versions.yaml

```diff
@@ -58,3 +58,7 @@ externals:
     description: "GNU parallel tool"
     url: "https://ftp.gnu.org/gnu/parallel"
     version: "20190222"
+
+  critools:
+    version: v1.14.0
+    openShift3Version: 1.0.0-beta.2
```

Comment:
We already encode openshift in the main versions database, so should these details go there?:

- https://github.com/kata-containers/runtime/blob/master/versions.yaml#L193
- https://github.com/kata-containers/runtime/blob/master/versions.yaml#L233

Reply:
Okay, I will add it there.

Reply:
Created another PR to address this issue: https://github.com/kata-containers/runtime/pull/1873